src/javaFirstAssessment/Person.java

```diff
@@ -0,0 +1,23 @@
+package javaFirstAssessment;
+
+public class Person {
+	protected String name; 
```

Comment:
- properly menggunakan private, karna untuk akses variabel nya nanti bisa lewat method getter dan setter
- variabel dengan tipe data yg sama bisa digabung dalam satu line
```suggestion
	private String name, address; 
```